faq.yaml

```diff
@@ -54,33 +53,34 @@ en:
+        simplifying the distribution mechanism.  However, it is still worth explicitly pointing out some
         of the issues with the policy. First, it places undue burden upon the launchers of the network to
         audit and decide on different generalized lock contracts, which may contain edge cases such as
         bypassing locking by issuing derivatives of the locked Ether (such as with WETH). Further, if a
         contract does not correctly lock, or if code claiming to be a lock bypasses detection through
         underhanded Solidity techniques, it could break the legitimacy of the initial distribution of
-        the entire network. Removing generalized locking removes the necessity for case-by-case decision-making
-        by a launch-team, and other edge case vulnerabilities.
+        the entire network. While Edgeware’s contract codes have been fully audited by Quantstamp,  Removing generalized
+        locking removes the necessity for case-by-case decision-making by a launch-team, and other potential edge case
+        vulnerabilities.
 
 
     - question: Were these faulty design decisions made in Edgeware to allow distribution to the victims of locked funds in the November 2017 Parity Multisig Exploit?
       answer: |
         The Web3 Foundation, the largest victim of the Parity bug, has a very close relationship
         with the Edgeware project, and it is likely it is due to the influence of the Web3 Foundation
         that these odd design decisions were made in the construction of the Edgeware lockdrop. But in
-        trying to right a wrong by recovering funds, design decisions were made that could threaten
-        the legitimacy of the project as a whole.
+        trying to right a wrong by recovering funds, design decisions were made that could hamper the effectiveness of the
+        project as a whole.
         
         Let’s be clear: Like many in the community, we feel for groups like the Web3 Foundation,
         and others affected by these attacks, but an open system of governance is one in which the
         people have a say, while a system in which rules are changed to provide select parties with
         powerful voting shares is not open. In the case of Edgeware, the potential for an open-future
         was impeded prior to launch by late-stage changes to a token distribution model that aimed to
-        artificially provide favor to a small, select group of individuals and teams. The result would
-        provide a substantial voting-percentage of all network-tokens to a party or parties not in control
-        of the funds for which tokens are rewarded.
+        provide favor to several small, select group of individuals and teams. While the changes were disclosed before the
+        lockdrop began The result would provide an estimated 9% substantial voting-percentage of all network-tokens to a
```

Comment:
Typo: "began The result" doesn't need a capital T

Challenge: how is this 9% calculated?

Reply:
> how is this 9% calculated?
 ++ @thomivy